pom.xml

```diff
@@ -44,6 +44,7 @@
         <module>providers/jdbc/shedlock-provider-jdbc-micronaut</module>
         <module>providers/mongo/shedlock-provider-mongo</module>
         <module>providers/mongo/shedlock-provider-mongo-reactivestreams</module>
+        <module>providers/neo4j</module>
```

Comment:
I would move this to a submodule. I assume there are multiple libraries you can use to access neo4j so we should make space for potential different implementation.

Reply:
I moved it to a subfolder with no additional intermediate maven submodule - now it's structured like the other folders.